apps/api/app.py

```
@@ -0,0 +1,128 @@
+"""FastAPI application for Taboot API with CORS, lifecycle management, and middleware."""
+
+import logging
+from contextlib import asynccontextmanager
+from typing import AsyncGenerator
```

Ruff (0.14.1)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)